.github/workflows/gen_pr.yaml

```diff
@@ -0,0 +1,54 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Run Generator and Create Pull Request
+on:
+  push:
+    branches:
+      - "master"
```

Comment:
Non-blocking: Filed #177 , I don't think it should block CI, but worth noting each thing added that specifies the `master` branch will make switching more challenging.